src/core/lib/gprpp/orphanable.h

```diff
@@ -133,18 +147,26 @@ class InternallyRefCountedWithTracing : public Orphanable {
 
   virtual ~InternallyRefCountedWithTracing() {}
 
-  void Ref() { gpr_ref(&refs_); }
+  RefCountedPtr<Child> Ref() {
```

Comment:
Likewise with must use result in all points here

Reply:
Done.  Also made the same changes in ref_counted.h.